CONTRIBUTING.md

```diff
@@ -1,11 +1,40 @@
-# How to Contribute
+# Contributing to Kubeflow Spark Operator
 
-We'd love to accept your patches and contributions to this project. There are
+Welcome to the Kubeflow Spark Operator project. We'd love to accept your patches and contributions to this project. There are
 just a few small guidelines you need to follow.
 
-## Code reviews
+## Follow the Code of Conduct
+
+Before any contribution you would like to make, please have a look at the [code of conduct](CODE_OF_CONDUCT.md), which details how contributors are expected to conduct themselves as part of the Kubeflow Spark Operator Community.
+
+## Code Contributions
+
+The Spark Operator project accepts code contributions via Github pull requests(PR), and this is the only way accepted to apply your changes to the Fluid project.
+
+You should submit a PR If any modifications is going to be applied to the Fluid project, including but not limited to:
+
+- Fix typos
+- Fix bugs
+- Fix or polish documents
+- Prune redundant codes
+- Add comments to codes for readability
+- Add missing test cases
+- Add new features or enhanced some feature
+- Refactor codes
+- ...
+
+## Signing Off Commits
+
+Kubeflow uses Developer Certificate of Origin ([DCO](https://github.com/apps/dco/)). You can sign off your contributions manually by doing ONE of the following:
+
+- Use `git commit -s ...` with each commit to add the sign-off.
+- Manually add a `Signed-off-by: Your Name <your.email@example.com>` to each commit message.
+
+The email address must match your primary GitHub email. You do NOT need cryptographic (e.g. gpg) signing.
+
+- Use `git commit -s --amend ...` to add a sign-off to the latest commit, if you forgot.
```

Comment:
Please redirect contributors to this page for DCO policies and explanation how to sign commits: https://www.kubeflow.org/docs/about/contributing/#sign-off-your-commits